bare/roles/web/tasks/nginx.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+- name: Copy nginx config
+  template: src=../files/nginx/mastodon.conf.j2 dest=/etc/nginx/sites-available/mastodon.conf
```

Comment:
Could we settle on either of the notations? Short form or long form, I don't really have a preference (although I've seen more long form than short form)

Reply:
The src is inside of ansible, the source is on the machine itself. I always did it this way to highlight this. Is there a way to notate a fixed path for the ansible folder? Open for comments, not an ansible expert here.

Reply:
Sorry, I think I framed it incorrectly .. I meant using

```
template:
  src: [...]
  dest: [...]
```

instead of the short form.